hnix-store-remote/src/System/Nix/Store/Remote.hs

```diff
@@ -210,7 +214,7 @@ buildDerivation p drv buildMode = do
     -- XXX: reason for this is unknown
     -- but without it protocol just hangs waiting for
     -- more data. Needs investigation
-    putInt 0
+    putInt (0 :: Int)
```

Comment:
I was keeping this one pop up in warnings intentionally as it's not clear why we need that.

Reply:
Yep, good catch. this is completely unrelated.